src/api-calls.js

```diff
@@ -29,12 +28,13 @@ const postBooking = (selectedRoom, customer, date) => {
       showSuccessMessage()
       fetchData()
     } else if (!response.ok) {
-      showErrorMessage()
+      throw new Error (`Error: ${respnse.body}`)
     }
-    console.log('post response', response)
-
   })
-  .catch(err => showErrorMessage())
+  .catch(err => {
+    console.log(err)
```

Comment:
Ooooh, if we have a function (like your **showErrorMessage()** ) is it necessary or is it redundant to have the console.log(err) there. I was about to say to remove it, but I see you've added it. Maybe I need to learn something here.